### ISSUE TITLE
image with text overlay

### ISSUE THREAD

**Issue by elfassy** [Semantic-Org/Semantic-UI#3459]
is it possible to do this?

<img src="http://semantic-ui.com/images/wireframe/image-text.png">

Where the text shows up above the image. Thanks

**Comment by kenhkelly** [comment]
Have you found a way to do this @elfassy? Looking for the same thing.

**Comment by elfassy** [comment]
@kenhkelly yes using regular css. Either negative margin or absolute positioning should work.

**Comment by kenhkelly** [comment]
I figured as much. Would be nice to have that added to the .ui.image element for simplicity.

**Comment by rvinay88** [comment]
👍 A pretty common pattern so would be nice to have this coded in by default.

**Comment by exogenesys** [comment]
@elfassy Could please tell us more about how you did it? Any idea how you could accomplish the same with the React Semantic UI 'Image'?

**Comment by paschalidi** [comment]
+1 please

**Comment by Tsourdox** [comment]
The "css way" would look something like this:

```
<div className="ui fluid image">
    <img src={url}/>
    <div style={{position: 'absolute', bottom: 0, width: '100%', height: 'auto'}}>TEST</div>
</div>
```

**Comment by paschalidi** [comment]
What if I want to do something like this 
```
<div className="ui fluid image">
    <img src={url}/>
    <div style={{position: 'absolute', bottom: 0, width: '100%', height: 'auto'}}>TEST</div>
    <div style={{position: 'absolute', bottom: 0, width: '100%', height: 'auto'}}>ANOTHER TEST</div>
</div>
```
I have the feeling that then the two divs will override each other, or ?

**Comment by mcclowes** [comment]
+1

**Comment by nrion** [comment]
I am having the same problem :(

**Comment by CloudMunk** [comment]
Why has this not been addressed yet? This is such a common thing to have in a UI library... Has anyone found an easy and sound workaround on this issue yet? The CSS way wont cut it for what I am doing currently.

**Comment by elfassy** [comment]
> Why has this not been addressed yet?

Feel free to open a PR.

**Comment by marcos-rios** [comment]
Any updates on this?

**Comment by kenhkelly** [comment]
Created a PR to add caption support: https://github.com/Semantic-Org/Semantic-UI/pull/6621

**Comment by lubber-de** [comment]
Supported since https://github.com/fomantic/Fomantic-UI/pull/247
